bsip-0076.md

```diff
@@ -0,0 +1,88 @@
+    BSIP: 0076
+    Title: Threshold for price feeds through voting
+    Authors: Abit More
+    Status: Draft
+    Type: Informational
+    Created: 2019-09-25
+
+# Abstract
+
+This BSIP defines a process to influence the pegging status of committee-owned smartcoins through poll voting.
+
+# Motivation
+
+In order to maintain the peg of the smartcoins, smartcoins holders now have the right to be always
+possible to buy collateral at a "fair" price. For committee-owned smartcoins, the collateral is BTS.
+However, when the BTS token is being dumped in Centralized Exchanges with fake volume and even fake supply,
```

Comment:
This 'fake volume/supply short attack' conspiracy has been outright rejected by ZB: https://twitter.com/ZB_Group/status/1188771828967206912?s=20

Reply:
Why? Debtors fully acknowledged this as a possible risk when they opened debt positions in **market pegged** assets. Let them GS instead of de-pegging.